OWML.Common/IModUnityEvents.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace OWML.Common
+{
+    public interface IModUnityEvents
+    {
+        void FireOnNextUpdate(Action action);
```

Comment:
Can add `event Action OnUpdate` and other Unity stuff here when/if needed later.

Reply:
Might as well do it now I suppose